q2_feature_table/_summarize/tabulate_seqs_assets/index.html

```diff
@@ -9,7 +9,7 @@
         button on the resulting page.
       <p>
         To download a raw FASTA file of your sequences, click
-        <a href="sequences.fasta" download="sequences.fasta">here</a>.
+        <a href="sequences.fasta" target="_blank">here</a>.
```

Comment:
I'm don't this really applies here as we're not opening a document (which means no `window.opener` api to exploit), it's a fasta file. Happy to add it though.